app/exawind/exawind.cpp

```diff
@@ -147,6 +149,7 @@ int main(int argc, char** argv)
         })) {
         exawind::NaluWind::initialize();
     }
+    sim.set_nw_start_rank(nalu_start_rank);
```

Comment:
Maybe, instead of `sim.set_nw_start_rank` we can just `sim.add_nw_start_rank(start)` and place that in the for loop to avoid creating `nalu_start_rank` here and then copying it in? And then `add_nw_start_rank` would be a `push_back` on `m_nw_start_ranks`.

Reply:
So the issue here is that the instance for `OversetSimulation` is not created until after the sub communicators have been created for each solver.

Reply:
ah. never mind then.